packages/engine/src/templates/root/index.ts.handlebars

```diff
@@ -46,25 +57,22 @@ export const EmbraceSQLEmbedded = async (rootContext ?: InternalContext) : Promi
     {{/eachInMap}}
     {{/eachInMap}}
   };
-  // map of our read only methods
+  // map of our read only methods -- this is used to figure out GET/POST
```

Comment:
Get rid of this second map and have a single map of EntryPoints — so we are clear that this is the far edge. Entry points will:
- have an executor method that takes a context, runs, returns context
- wrap in an overall transaction and auto commit on a clean finish and auto abort on dirty finish
- read only indicator